database/gameUser.go

```diff
@@ -18,12 +18,12 @@ const (
 )
 
 type GameUser struct {
-	ID        string `gorm:"unique"`
+	ID        string
 	Email     string
-	Username  string
+	Username  string `gorm:"primaryKey"`
 	Password  string
 	CreatedAt time.Time
-	GameID    string
+	GameID    string `gorm:"primaryKey"`
```

Comment:
Why do we have 2 primary keys in this struct? 

GameId should be a secondary key here and a primary key in another struct.

Reply:
By doing this we create a composite key par og the username and gameID meaning that the combination of username+gameID should be unique meaning that we can have a "Lanestolen" user in each game
At the moment we dont store games in the database